client/src/controllers/scoresheet/scoresheet_controller.js

```diff
@@ -87,13 +97,24 @@ class ScoresheetController {
         if (scoreDiff !== 0 && isFinite(scoreDiff)) {
             this.showScoreDiffAnimation(scoreDiff)
         }
+
+        if (this.match && this.match.indexOf('✔') > -1) {
+            return '"' + newScore.toString() + '"'

```

Comment:
1. Why are we adding quotes around a score for an existing match?
2. You should use string templates here, like this: \`"${newScore.toString()}"\`

Reply:
1. To signify that this is a previously completed match.